### DISCUSSION TITLE
[Nhận task]

### DISCUSSION THREAD

**Discussion by trungdoublelift** [Runway-Club/Runway-CV-Builder#65] [Task Delivery]
Chức năng thêm xoá sửa trong CV List

**Comment by nguyentantrung2000** [comment]
Thông tin task:
Thao tác thêm xoá sửa của User trong CV List 
Yêu cầu:
Phải xoá được , và thêm đc CV theo ý của user
Deadline 11:59PM 12/8/2021

**Reply by trungdoublelift** [reply]
OK
